Clarify difference between harmonised and referenced standard (#188)
orcwg/cra-hub: faq.md

### Standards

<details>

>     <a name="q-what-is-a-harmonised-standard-and-why-does-it-matter"></a>

Reviewer: That's to maintain a legacy link, so let's not touch it.
```suggestion
    <a name="q-what-is-a-harmonized-standard-and-why-does-it-matter"></a>
```

<details>
    <a name="q-what-is-a-harmonized-standard-and-why-does-it-matter"></a>
    <summary><strong><a name="faq-tmp-56" href="#faq-tmp-56">tmp-56.</a> What is a <em>harmonised standard</em> and why does it matter?</strong></summary>

> A _harmonised standard_ is a standard adopted by one of the [European Standardisation Organisations (ESOs)][ESO]. Certain (but not all) _harmonised standards_ are referenced in the _Official Journal of the European Union_.

Reviewer 2: Can we add a link to the journal?

> Any harmonised standard  referenced in this way provides organisations that implement them with a _presumption of conformity_ with the law.

Reviewer: ```suggestion
Harmonised standards referenced in this way provide organisations that implement them with a _presumption of conformity_ with the regulation. Harmonised standards may be referenced with restrictions, in which case they only provide partial _presumption of conformity_.
```

Author: I do not think this is entirely correct  as it also applies to directives (see e.g. https://eur-lex.europa.eu/eli/dec_impl/2025/138/oj).
The NLF recitals say "The presumption of conformity to a legal provision conferred by conformity to a harmonised standard should enhance recourse to compliance with harmonised standards."

Author: See [`c2e9082` (#188)](https://github.com/orcwg/cra-hub/pull/188/commits/c2e9082ede5830eccc567e24bd17c3b61293d4a2)

> This is why it is expected that most organisations will choose to implement these referenced _harmonised standards_ to comply with the CRA.

Reviewer: ```suggestion
The _presumption of conformity_ provided by harmonised standards referenced in the _Official Journal of the European Union_ is why it is expected that most organisations will choose to implement such standards when they exist in order to comply with the CRA.
```

However, not all harmonised standards are referenced.

> Those that are not referenced are often foundational standards upon which other standards build.

Reviewer: ```suggestion
Those that are not referenced are often foundational standards upon which other standards build. In general, only the vertical (product-specific) standards are referenced, though sometimes horizontal standards that cover generic requirements may be referenced with restrictions.
```

Author: They also may be referenced without restrictions...

> For example, vertical standards (industry-specific standards) may build upon horizontal standards (general standards that apply across industries).

Reviewer: ```suggestion
```

The ORC WG maintains a [list of _harmonised standards_][standards] requested by the European Commission to the ESOs.

> This list indicates whether they are going to be referenced and thus provide a presumption of conformity.

Reviewer: ```suggestion
```

Author: See [`c2e9082` (#188)](https://github.com/orcwg/cra-hub/pull/188/commits/c2e9082ede5830eccc567e24bd17c3b61293d4a2)
